test/Swashbuckle.AspNetCore.Annotations.Test/ModelMetadataHelper.cs

```diff
@@ -19,6 +25,7 @@ public static IModelMetadataProvider GetDefaultModelMetadataProvider()
                     new DataAnnotationsMetadataProvider(Options.Create(new MvcDataAnnotationsLocalizationOptions()), null),
                 }
             ));
+#endif
         }
     }
 }
```

Comment:
^^^ This file can just be deleted as it's deadwood, not referenced anywhere

Reply:
Can you elaborate on that ? that's really the part of the code i'm loosing track, i've just used the `Empty` one for now in order to makes the code bui;d (PR still in Draft).
52 test are still KO because something is this class does not generate proper fake data for 3.0 ;) not sure what yet

Reply:
I'm just saying the entire file `ModelMetadataProvider` can be deleted entirely (as opposed to modified) as it's no longer in use.

Reply:
oO right, thx, deleted